File: Inheritance/Date.cpp
#include<iostream>
using namespace std;
#include"Date.h"

Date::Date()
{
    day=month=year=1;
}
Date::Date(int d,int m,int y)
{
    day=d; month=m; year=y;
}
void Date::display()
{
 cout<<day<<"/"<<month<<"/"<<year;
}

File: java/OOP2/Date.java
class Date
{
  private int dd,mm,yy;
  public void showDate()
   {	
     System.out.println(dd+"/"+mm+"/"+yy);
   }
   public void showDate(char ch)
  {
   System.out.println(dd+""+ch+mm+ch+yy);
  }
  Date()
   {
	dd=0;mm=0;yy=0;
   }
  Date(int dd,int mm,int yy)
  {
     this.dd=dd;this.mm=mm;this.yy=yy;
  }
  Date(Date d)
  { 
    this.dd=d.dd;
    this.mm=d.mm;
    this.yy=d.yy;
  }
}
